src/posts/blog/6-masks/masks.ts

```diff
@@ -0,0 +1,277 @@
+* **Breathability:** Good
+* **Useful for:** Any regular indoor public spaces -- shopping, transit
+* **Benefits:** Good enough protection for most scenarios  
+* **Downsides:** Can’t get wet
+
+<figure>
+<img src=${n95} width="296", height="329"/>
+</figure>
+
+N95 respirators are the gold standard for healthcare workers.  Back in the early days of COVID, hospitals were desperate for them so that doctors and nurses could be adequately protected as they tended to COVID patients.  Now, thanks to some very aggressive efforts by companies around the world to increase supply, it’s easy for anyone who wants an N95 respirator to get one.  As a result, you should always wear an N95 if you’re going to an indoor public space.
+
+Why should you wear an N95 when indoors?  Because they’re much, *much* better than cloth masks.  In an average part of America right now, just going to the grocery store with a cloth mask can give you a 1 in 3,000 chance of getting COVID.  Switching to an N95 mask drops that risk by 8x, to 1 in 24,000.  Interestingly, some of the countries that have most successfully fought COVID (e.g., Taiwan) have mailed free N95s to all their residents.  
+
+However, in order to actually get that full benefit, you need to do some things properly:
+
+
+### Quality
+*   Look for “NIOSH-approved” on the product.  This means that the N95 has been properly tested for filtration efficiency by a government standards organization. 
+
+### Fit
+*   You need to get an N95 mask that fits your face properly.  Even small holes around the side will drop the mask’s filtration efficiency from 95% to something like 70% ([link to study](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3306645/)).  Thus it’s vital that your mask forms a tight seal around your mouth.  You may need to try a few brands until you find one that fits you and your beautiful, unique nose.  [Here’s a video guide to mask handling and seal checks](https://www.youtube.com/watch?v=U8Inww-1avg) (note that the video shows them using valved respirators, but this advice also applies to valveless respirators).  Basically, when you breathe in or out sharply, you should not feel any air sliding in around the edge of the mask. If you have a beard or other facial hair that interferes with the mask’s ability to form a seal, you DO NOT have a good fit.  While it’s still good to wear an N95, you are only getting moderately better protection than a surgical mask if you have facial hair.  We're sure your beard is sexy, but not giving your friends COVID is even sexier! 
+*   The nose bridge and the bottom of the chin are where fit is most often an issue.  Any good N95 will have a wire or metal strip on the nose bridge so that it will fit your nose.
+*   The mask should have straps that go around the back of your head, not just ear loops.  Head straps give you a tighter fit
+
+### Reusing your N95
+*   You’ll need to throw out your N95 mask if it gets wet, overly dirty, or difficult to breathe through.
```

Comment:
Could this be worded less aggressively? A boiled N95 mask is still better than anything earlier on this list.

Reply:
Yeah I can do that.